bin/revert

```diff
@@ -39,14 +40,17 @@ def post_comment(
         repo_id: int,
         pr: int,
         revert_sha: str,
+        reverter_username: str | None,
         headers: dict[str, str],
 ) -> None:
-    data = {'body': f'PR reverted: {revert_sha}'}
+    message = f"PR reverted: {revert_sha}."
+    if reverter_username:
+        message += "\n\n@{reverter_username} Did you leave a comment describing why this PR was reverted?"
```

Comment:
there's no guarantee that the username matches the git user so this might at mention some rando

Reply:
I could add an additional option to pass the username of the person reverting the label, then we can drop the regex and it's more explict?

Reply:
Since right now we do actually pass the login name of the user https://github.com/getsentry/sentry/blob/14a29a1ed05985d5a629524bca3cbbd72360347c/.github/workflows/fast-revert.yml#L28

Reply:
maybe but it feels like we're trying to use tech to solve a culture problem.  imo we shouldn't need this and it's just going to be annoying (more notification noise) to those of us that are already doing this correctly

Reply:
I think this is helpful - it's easy to forget to do this. Is there an easy way we could add an opt-out for folks that don't want it?